### ISSUE TITLE
Invalid package name 

### ISSUE THREAD

**Issue by subdesign** [franzliedke/studio#8]
Hi,
Trying to create a new package on a fresh/latest L5 instance, got error:
```php
user@user:/var/www/laravel5> vendor/bin/studio create testpackage
Please name this package test
 Invalid. Try again. 
Please name this package testpackage
 Invalid. Try again. 
Please name this package Hello
 Invalid. Try again. 
Please name this package 
 Invalid. Try again. 
```
studio version 0.9.3
What do I make wrong?

**Comment by franzliedke** [comment]
Hmm, I guess I have to improve the error message, eh?

The package name should be a valid Composer package ID, something along the lines of `vendor/name`.

**Comment by subdesign** [comment]
Thanks, that way worked, cheers

**Comment by franzliedke** [comment]
Wonderful.

I'll leave this open for now as a reminder to improve the error messages.

**Comment by franzliedke** [comment]
The error messages are now improved in the just-released [0.9.4 release](https://github.com/franzliedke/studio/releases/tag/0.9.4) - please update.

And thanks for the input!
